### PR TITLE
reflect director-free; update outdated links

### PR DESCRIPTION
**Pull request by xueyuanjia**
Fixes https://github.com/w3c/Guide/issues/184

### CODE REVIEW

File: process/charter.html [hunk 1]
```diff
@@ -253,11 +253,11 @@ <h4>Updating the Charter</h4>
 <h2>5. <a id="cfr" name="cfr">Advisory Committee Review of the
 Charter</a></h2>
 
-<p>Per <a href="https://www.w3.org/Consortium/Process/#CharterReview">section 5.2.3
-of the W3C Process</a>, The Director must solicit Advisory Committee review of every new or substantively modified Working Group or Interest Group charter.
-The Director is not required to solicit Advisory Committee review prior to a charter extension or for minor changes. The review period must be at least 28 days.
+<p>Per <a href="https://www.w3.org/2023/Process-20230612/#CharterReview">section 4.3
+of the W3C Process</a>, The Team must solicit Advisory Committee review of every new or substantively modified Working Group or Interest Group charter.
+The Team is not required to solicit Advisory Committee review prior to a charter extension or for substantive changes to a charter that do not affect the way the group functions in any significant way. The review period must be at least 28 days.
```
Comment (plehegar) [inline process/charter.html:258]: What is the rational for allowing the Team to make "substantive changes to a charter that do not affect the way the group functions in any significant way" ? It used to say "for minor changes" and that is all that the Team can do.

**Review by plehegar — CHANGES_REQUESTED** [review]
One change seems incorrect

File: process/charter.html (hunk 1 above)
Comment (xueyuanjia) [reply process/charter.html:258]: "for minor changes" was the description in Process 2017-2019 ([section 5.2.3](https://www.w3.org/2017/Process-20170301/#CharterReview)). Starting Process 2020 ([section 5.2.3](https://www.w3.org/2020/Process-20200915/#CharterReview)) and then the present Process 2023 ([section 4.3](https://www.w3.org/2023/Process-20230612/#CharterReview)), the wording had been changed to "substantive changes to a charter that do not affect the way the group functions in any significant way".

Comment (plehegar) [reply process/charter.html:258]: oh, good catch!

**Review by plehegar — APPROVED** [review]

**Pull request merged**